package.json

```diff
@@ -252,16 +259,16 @@
     "react-icons": "4.2.0",
     "react-popper": "2.2.5",
     "react-query": "3.13.4",
-    "react-redux": "7.2.3",
+    "react-redux": "7",
```

Comment:
this has been downgraded? so was react? 🤔

Reply:
This was an attempt at fixing the types between react & redux while ensuring we're using the most recent version possible. Specifying `7` here will install the latest version 7.x.x available, and same with react's version `17`. The types for react are fixed at `17.0.30` which works for both packages, which I got from [this post](https://github.com/facebook/react/issues/24304#issuecomment-1092563688). If it looks cleaner, can just set the exact version that's now in yarn.lock

Reply:
As discussed offline, will pin the version numbers to avoid circumventing version checks and to follow security audit recommendations.